external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutTopicPartition.java

```diff
@@ -43,7 +43,7 @@ public TopicPartition getTopicPartition() {
 
     @Override
     public String getId() {
-        return topicPartition.topic() + "/" + topicPartition.partition();
+        return topicPartition.topic() + "_" + topicPartition.partition();
```

Comment:
Yes it will work if no one is willing to extract the information via parsing the return value. OK to leave as it is, but if possible I'd like to avoid the possible confusing case. 

If you're worrying about creating redundant znode because of '/', how about choosing the character which is not legal char for kafka topic but legal char for znode name, for example, `@`?